libsplinter/src/admin/service/error.rs

```diff
@@ -22,6 +22,68 @@ use crate::signing;
 
 use protobuf::error;
 
+#[derive(Debug)]
+pub enum AdminServiceError {
+    ServiceError(ServiceError),
+
+    GeneralError {
+        context: String,
```

Comment:
I like this pattern for error handling. What would a context string look like? Is it just an error message.

Reply:
You can see two examples here: https://github.com/Cargill/splinter/blob/8d3ef6f7dfd47ee19df470d2a783a592b0f8607f/libsplinter/src/admin/service/mod.rs#L369-L381